### ISSUE TITLE
Evaluation Loss

### ISSUE THREAD

**Issue by H0key-22** [Bowen-Sun-0728/Blockwise-SFT#2]
Hi Bowen ! Thanks for the great insight in the alignment between training and inference.
I recently tried the finetuning of gsm8k on Llada model and the experiment show kind of overfitting, where the training loss drops and eval loss maintain. I guess the Llada might be trained already on gsm8k.
Did you record the eval(val) loss during the blockwise training process?

**Comment by Bowen-Sun-0728** [comment]
Thanks for your interest. 
I am not sure whether GSM8K is in the pre-traning or post-training data of LLaDA. If the behavior on GSM8K is significantly different from other dataset, your assumption is very persuasive. 
Besides, the eval loss are very similar with training loss during the reported steps, or we would have stopped fine-tuning before that happens.
